hudi-flink/src/test/java/org/apache/hudi/utils/TestStreamerUtil.java

```diff
@@ -70,5 +74,24 @@ public void testInitTableIfNotExists() throws IOException {
             .build();
     assertFalse(metaClient2.getTableConfig().getPartitionColumns().isPresent());
   }
+
+  @Test
+  void testMedianInstantTime() {
+    String higher = "20210705125921";
+    String lower = "20210705125806";
+    String median1 = StreamerUtil.medianInstantTime(higher, lower);
+    assertThat(median1, is("20210705125843"));
+    // test symmetry
+    String median2 = StreamerUtil.medianInstantTime(lower, higher);
```

Comment:
If the order is reversed, I think we should throw an error? Otherwise, this may hide a bug

Reply:
Agree, thanks.